rabbitmq/os_tweaks.sls

```diff
@@ -0,0 +1,11 @@
+{% from "rabbitmq/map.jinja" import rabbitmq with context %}
+
+increase_file_descriptor_limit:
+  cmd.run:
+    - name: sysctl -w fs.file-max={{ rabbitmq.fd_limit }}
+  file.append:
+    - name: /etc/sysctl.conf
+    - text: fs.file_max={{ rabbitmq.fd_limit }}
```

Comment:
Is this for setting the max to be high due to rabbitmq potentially open streams for a large number of files?